HPXMLtoOpenStudio/resources/schedules.rb

```diff
@@ -1350,6 +1350,7 @@ class SchedulesFile
   ColumnBattery = 'battery'
   ColumnBatteryCharging = 'battery_charging'
   ColumnBatteryDischarging = 'battery_discharging'
+  ColumnMaximumPowerRatio = 'maximum_power_ratio'
```

Comment:
I think it's important to include HVAC in the name: `hvac_maximum_power_ratio`. Don't forget to update docs.